components/config/definition.rst

```diff
@@ -212,6 +212,9 @@ Before defining the children of an array node, you can provide options like:
 ``addDefaultsIfNotSet()``
     If any child nodes have default values, use them if explicit values
     haven't been provided.
+``normalizeKeys(false)``
+    If called (with ``false``), keys with dashes are *not* normaled to underscores.
+    It is recommended to use this to avoid this unnecessary normalization.
```

Comment:
only for prototyped nodes (and when it is a map, as it does not make sense for a list where keys are numeric)